package.json

```diff
@@ -56,6 +58,7 @@
     "@vitejs/plugin-react": "^2.2.0",
     "@walletconnect/types": "2.11.0",
     "autoprefixer": "^10.4.14",
+    "dotenv": "16.3.1",
```

Comment:
Why was this added? I see the `require('dotenv').config();` line was removed below

```suggestion
```